pages/models/generations/GENTPJ/index.md

```diff
@@ -0,0 +1,233 @@
+
+For the GENTPF/GENTPJ, generally $$ X_d'' <> X_q''$$, which result in some more complex equations, but it allows to consider the impact of the transient saliency. The following expressions are used to calculate the saturated values of the reactances:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ X_{dsat}'' = \frac{X_d'' - X_l}{Sat_d} + X_l $$
+$$ X_{qsat}'' = \frac{X_q'' - X_l}{Sat_q} + X_l $$
+
+</div>
+
+#### Network interface equations
+
+The following circuit equations were obtained in the GENSAL model:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ \frac{1}{\omega_s} \frac{d\Psi_d}{dt} = R_s I_d + \frac{\omega_m}{\omega_s} \Psi_q + V_d$$
+$$ \frac{1}{\omega_s} \frac{d\Psi_q}{dt} = R_s I_q - \frac{\omega_m}{\omega_s} \Psi_d + V_q$$
+
+</div>
+
+With proper transformations using the expressions for the fluxes, they can be rewritten as: 
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ V_d = - R_s I_d +  X_q''I_q + E_d''\frac{\omega_m}{\omega_s}$$
+$$ V_q = - R_s I_q -  X_q''I_d + E_q''\frac{\omega_m}{\omega_s}$$
+</div>
+
+with  $$E_d'' = E'_d \frac{X_q'' - X_l}{X_q' - X_l} + \Psi_q'' \frac{X_q' - X_q"}{X_q' - X_l} $$, and $$E_q'' = E'_q \frac{X_d'' - X_l}{X_d' - X_l} + \Psi_d'' \frac{X_d' - X_d"}{X_d' - X_l} $$, and where the scaling $$\frac{\omega_m}{\omega_s}$$ is ommited for the subtransient reactances. As discussed for the GENSAL model, $$X_d'' = X_q''$$, which allows to compact the grid interface equations in a single equation:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ V_d + jV_q = \frac{\omega_m}{\omega_s} (E_d'' + jE_q'') - (R_s + jX_d'') (I_d + jI_q) $$
+
+</div>
+
+This is not doable in the GENTPJ model, since generally the subtransient reactances are not equal. In this case, the saturated values of the subtransient reactance are used:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ V_d = \frac{\omega_m}{\omega_s} E_d'' + I_q X_{qsat}'' - I_dR_a$$
+$$ V_q = \frac{\omega_m}{\omega_s} E_q'' - I_d X_{dsat}'' - I_qR_a$$
+
+</div>
+
+Again, the frequency factor can be added to the term $$I_q X_{qsat}''$$ and $$I_d X_{dsat}''$$, but it is commonly omitted in transient stability studies.
+
+The following figure shows the block diagram for this GENTPJ model:
+
+<p align="center">
+<img src="{{'/pages/models/generations/GENTPJ/GENTPJ_blocks.png' | relative_url }}"
+     alt="Block diagram of the GENTPJ model"
+     style="float: center; width: 600px;" />
+</p>
+<div style="text-align:center">
+Figure 1: Block diagram of the GENTPJ model
+</div>
+<br>
+
+## Open source implementations
+
+No open-source implementations were found.
+
+
+## Table of references
```

Comment:
Please refer to the model template and add as much as information possible to tables of references

http://127.0.0.1:4000/pages/templates/modelTemplate.html

![image](https://github.com/user-attachments/assets/6141d919-1aa2-4138-b9c9-5f215f64bb90)

Reply:
Improved the references